src/libraries/System.Numerics.Tensors/src/System/NIndex.cs

```diff
@@ -91,7 +91,16 @@ public static NIndex FromEnd(nint value)
             return new NIndex(~value);
         }
 
+        /// <summary>
+        /// Convert the NIndex to an Index.
+        /// </summary>
+        /// <returns>The converted <see cref="Index"/></returns>
```

Comment:
```suggestion
        /// <returns>The converted <see cref="Index"/>.</returns>
```

Reply:
In general you don't need to include the type in parameter and return descriptions. It's shown automatically in IntelliSense and docs.